src/Serialization/Json/OkContentNormalizer.php

```diff
@@ -56,7 +56,12 @@ public function normalize($object, $format = null, array $context = array())
 
         // Pagination case
         if ($content instanceof Pagerfanta) {
-            $result['meta'] = ['totalPages' => $content->getNbPages()];
+            $result['meta'] = [
+                'countPages'  => $content->getNbPages(),
+                'countResult' => $content->getNbResults(),
+                'currentPage' => $content->getCurrentPage(),
+                'maxPerPage'  => $content->getMaxPerPage(),
```

Comment:
This is out of apijson specs. But makes a lot of sense. 👍

Reply:
But `totalPages` and `totalResult` aren't better names?

Here are pros:
1. Keeping `totalPages` does not breaks compatibility
2. [Hydra](https://github.com/HydraCG/Specifications/blob/master/drafts/use-cases/3.2.pagination.md) also uses `total` wording for this kind of stuff

Reply:
Yes totally my bad, I just wrote that on my current project, so I didn't take care about BC. ^^'